packages/hardhat-core/src/internal/cli/constants.ts

```diff
@@ -1 +1 @@
-export const SUPPORTED_NODE_VERSIONS = ["^18.0.0", "^20.0.0", "^22.0.0"];
+export const SUPPORTED_NODE_VERSIONS = ["^20.0.0", "^22.0.0", "^24.0.0"];
```

Comment:
```suggestion
// TODO: Remove 18 on June 30, 2025
export const SUPPORTED_NODE_VERSIONS = ["^18.0.0", "^20.0.0", "^22.0.0", "^24.0.0"];
```

According to [this](https://hardhat.org/hardhat-runner/docs/reference/stability-guarantees) we still need to support it. Other than that, this looks good.

Reply:
I guess there are some tests below that would need to be updated

Reply:
No need to read 18 to the ci though